VulnerableApplication/Global.asax.cs

```diff
@@ -10,12 +10,18 @@ namespace VulnerableApplication
 {
     public class MvcApplication : System.Web.HttpApplication
     {
+        public string Password = "abcdef";
+    
```

Comment:
[CODE_SMELL] 8 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.